File: 7kyu/js/reversed-words/reversed-words.js
function reverseWords(str) {
  var reversed_words = [];
  var split_space = str.split(' ');
  for (var k of split_space){
    var reversed = '';
    for (var j of k){
      reversed = j + reversed;
    }
    reversed_words.push(reversed);
  }
  return reversed_words.join(' ');
}

File: 7kyu/python/reversed-words/reversed_words.py
"""Reverses each words."""


def reverse_words(text):
    """Reverse words."""
    result = []
    space_split = text.split(' ')
    for i in space_split:
        reversed = ''
        for k in i:
            reversed = k + reversed
        result.append(reversed)
    return ' '.join(result)


def other_reverse_words(str):
    """Best practices."""
    return ' '.join(s[::-1] for s in str.split(' '))
